### ISSUE TITLE
cub.createGeoTiff() and cub.saveGEOJSON() not working

### ISSUE THREAD

**Issue by astroice7** [seignovert/pyvims#4]
Hello again @seignovert,

I would definitely like to utilize the VIMS data as GeoTIFFs for polar projections and recreate the maps from the VIMS website, but I run into the following error: `OSError: Kernel folder "kernels/" does not exist` when I try to generate the GeoTIFF or GeoJSON file with the cookie cutter code below:
```
# download data
wget.download("https://vims.univ-nantes.fr/data/isis/003TI/C1487096932_1_vis.cub")
wget.download("https://vims.univ-nantes.fr/data/isis/003TI/C1487096932_1_ir.cub")
wget.download("https://vims.univ-nantes.fr/data/isis/003TI/N1487096932_1_vis.cub")
wget.download("https://vims.univ-nantes.fr/data/isis/003TI/N1487096932_1_ir.cub")
wget.download("https://vims.univ-nantes.fr/lbl/1487096932_1", out="v1487096932_1.lbl") 
wget.download("https://vims.univ-nantes.fr/qub/1487096932_1", out="v1487096932_1.qub")
# run respective functions
testcub = pyvims.VIMS("1487096932_1")
testcub.createGeoTiff()
testcub.saveGEOJSON()
```

I tried looking into the source code to spot any obvious errors, but it must be linked to this _\kernels_ folders not being created with the pyvims SPICE routines for some reason. I tried to download the raw .qub and .lbl files in the same folder, but I still received the same error. Thanks for taking the time to look into this issue.

**Comment by seignovert** [comment]
Hi again @astroice7,

I see that you dive a bit into the code (which is perfectly understandable since the documentation is still an ongoing process…).
The `saveGeoTiff` and `saveGEOJSON` functions are very experimental. They work for a large number of cubes but require more testing to be fully efficient (its on my todo list).
When completed, theses products will be added to the VIMS website directly in addition to the `.cub` files.

However, you can already use these functions if you want. To do so, you need to have [`spicypy`](https://github.com/AndrewAnnex/SpiceyPy) installed. Then download at least theses kernels files into a `kernels/` folder inside your working directory:
* [Leap-second](https://naif.jpl.nasa.gov/pub/naif/pds/data/co-s_j_e_v-spice-6-v1.0/cosp_1000/data/lsk/naif0012.tls)
* [Spacecraft clock](https://naif.jpl.nasa.gov/pub/naif/pds/data/co-s_j_e_v-spice-6-v1.0/cosp_1000/data/sclk/cas00171.tsc)
* [Planetary Constants](https://naif.jpl.nasa.gov/pub/naif/pds/data/co-s_j_e_v-spice-6-v1.0/cosp_1000/data/pck/pck00010.tpc)
* [Spacecraft and Planet Ephemeris](https://naif.jpl.nasa.gov/pub/naif/pds/data/co-s_j_e_v-spice-6-v1.0/cosp_1000/data/spk/171215R_SCPSEops_97288_17258.bsp)

(This list can be found in the [`spice_cassini.py`](https://github.com/seignovert/pyvims/blob/master/pyvims/spice_cassini.py#L6-L19) file)

With them, you should be able to unlock these functions but be aware that it is an experimental feature which could evolve significantly in the future.

I keep the issue open, until this feature is completely implemented.

**Comment by seignovert** [comment]
It looks like the `Spacecraft and Planet Ephemeris` link is broken on the latest kernels release in October 2018…
I will need into look to this more carefully.

**Comment by astroice7** [comment]
Oh now I see why this is the beta version and the Jupyter notebook was limited. I appreciate you giving an updated status on this library. I am planning to these VIMS datasets for research on Titan, but map making is not a top-priority now.

I tried to download these files and just use the most recent `Spacecraft and Planet Ephemeris` for Sept. 2018 and edit the `spice_cassini.py` accordingly to see if it worked for VIMS cubes `1487096932_1` & `1781811236_1`. However, I had no luck and received the following `SpiceyError` from line 59 of `spiceypy.py`:
```
Toolkit version: N0066

SPICE(SPKINSUFFDATA) --

Insufficient ephemeris data has been loaded to compute the 
state of -82 (CASSINI) relative to 0 (SOLAR SYSTEM 
BARYCENTER) at the ephemeris epoch 2014 JUN 18 18:39:01.830.

subpnt_c --> SUBPNT --> SPKEZP --> SPKSSB --> SPKGEO
```
Hopefully, this might provide some help in working out the kinks of the experimental GeoTIFF and GeoJSON functions. As a heads up, I was able to generate the ArcMap .xml and ENVI .hdr files though. No worries if it will take some time to evaluate this issue.
